screens/HomeScreen.js

```diff
@@ -0,0 +1,53 @@
+import {React, useState} from "react";
+
+import { View, Text, Image, TouchableOpacity } from "react-native";
+import Container from "../common/Container";
+import EStyleSheet from "react-native-extended-stylesheet";
+
+
+function HomeScreen({ navigation, route }) {
+
+    return(
+        <Container>
+            <View style={styles.profile}>
+                <TouchableOpacity onPress={() => navigation.navigate("Profile", route.params)}>
```

Comment:
Instead of making a whole view for this button, use absolute positioning to position it at the top right of the screen. This way, it will always be there regardless of whether the screen below it (that will eventually be there) moves. Style looks something like this:
button: {
    position: 'absolute',
    right: '5%',
    top: '5%',
}